TundraTD/Assets/Scripts/Mobs/MobsBehaviour/MobBehaviour.cs

```diff
@@ -16,6 +18,11 @@ public abstract class MobBehaviour : MonoBehaviour
         [SerializeField] private GameObject[] effectPrefabs;
         [SerializeField] private MobModel mobModel;
         [SerializeField] private WayPoint[] waypointRoute;
+        
+        public bool IsMobMoving { get; set; }
```

Comment:
Иногда описание свойств выглядит слишком очевидным. Нет смысла писать такие тонны текста
```suggestion
        public bool IsMoving { get; set; }
```

Reply:
Исправил